src/bp/nlu-core/training-pipeline.ts

```diff
@@ -244,10 +257,12 @@ const TrainIntentClassifier = async (
         }
       ])
       .flatMap(i =>
-        i.utterances.map(utt => ({
-          label: i.name,
-          coordinates: getIntentFeatures(utt, customEntities)
-        }))
+        i.utterances
+          .filter(u => !u.augmented) // we don't want to train on augmented utterances as it would slow down training too much
```

Comment:
Don't you think there actually is semantic value in the examples? 

If we have an intent `buy-fuits` with one variable of type `Fruits` (which is a list);
The examples fruit examples in utterances (`buy a [banana](fruit_to_buy) now`) contain information.

As I understand, all we give to the svm right now is `buy a $fruit_to_buy now` which is quite poor compared to what we give on branch `dev`.

Am I in the potatos?